assets/questions/i18n.json

```diff
@@ -1,6 +1,10 @@
 {
   "interview_question": {
     "title": "Internationalization (i18n)",
+    "starter_code": "def abbreviate(word):\n\treturn \"\"\n\ndef are_all_unique(words):\n\treturn True",
+    "auxiliary_code": {
```

Comment:
Don't forget to add the dict to starter_code, here and in both places in the other file.